sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -501,6 +501,7 @@ export interface ServiceBusReceiver {
 // @public
 export interface ServiceBusReceiverOptions {
     identifier?: string;
+    keepDateType?: boolean;
```

Comment:
Not quite happy with the name, maybe something like `parseDates` or something to indicate they're being turned into an object?

Reply:
The option is more about preserving the original Date objects than parsing a number/string into Date object.  Maybe "preserveDates"?

Reply:
Something along the lines of `skipDateToNumberConversion`?

Reply:
Or `skipConvertingDateToNumber` to be more consistent with the other option `skipParsingBodyAsJson`

Reply:
`skipConvertingDate`? parsing and converting.

Reply:
+1 on `skipConvertingDate`.

Reply:
@xirzec any thoughts on the latest proposed name?

Reply:
looks good to me!